frontend/src/pages/Login.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react";
+import { Formik, Form } from "formik";
+import { useTranslation } from 'react-i18next';
+import axios from "axios";
+import MockAdapter from "axios-mock-adapter";
+import * as Yup from 'yup';
+
+import Button from "../components/Button";
+import InputField from "../components/InputField";
+
+const mock = new MockAdapter(axios)
+
+mock.onPost("http://localhost:3000/api", { email: "teszt@teszt.com", password: "1234567" })
```

Comment:
A mockolás is lehetne egy helyen egy külön fileban, amit mondjuk az API absztrakciót rejtő file húzna be.